### DISCUSSION TITLE
6月28日(土)OSC2021 Hokkaido/Onlineにてセミナー開催

### DISCUSSION THREAD

**Discussion by ykominami** [toppers/users-forum#24] [General]
6月28日(土)に開催されるOSC2021 Hokkaido/Onlineにてセミナーを行います。
12:00-12:45 C会場 「高品質な組込みOSを開発するTOPPERSプロジェクトのご紹介 ～はじめて使う人のためのマイコンボード、開発環境の選び方と使い方～」
担当：特定非営利活動法人TOPPERSプロジェクト
講師：堀 武司」
https://event.ospn.jp/osc2021-online-do/

OSCへの参加は、以下のURLより参加登録が必要です。
https://ospn.connpass.com/event/211390/

**Comment by Y-Ishioka** [comment]
知り合いから6/26(土)に開催されたOSCのTOPPERSのセミナー動画がyoutubeに上がらないのかって聞かれました。
知り合いは6/26は都合がつかず録画を後から見れるだろうからと思っていたようです。
以前のOSCの動画は公開されているので、TOPPERSプロジェクトとしてOKを出せば公開されるのかなと思うのですが、どうでしょうか？
情報は生ものだったりもするし、コンテスト募集の話しはかなり期間限定なので、公開されるなら早めがよいのではないでしょうか。
検討ください。

**Reply by ykominami** [reply]
Y-Ishiokaさんの問い合わせに先ほど気が付きました。遅くなってすみません。
確かに現状は、YouTube上に公開されていません。
セミナーの動画の公開についてきちんと確かめたうえで回答したいと思います。
よろしくお願いします。

**Reply by yurie** [reply]
こちらお待たせしてしまってすみません！
OSCさんの方から内容確認の依頼をいただいていて、担当多忙につき、時間が取れていない状況です。
公開までもう少しお時間ください。

**Reply by hori-takeshi** [reply]
OSC 出展担当者です。先ほど、OSC 2021 北海道のサイトで資料と動画が公開されました。
遅くなってしまってすみません。

**Reply by Y-Ishioka** [reply]
連絡ありがとうございます。
これですね。
<https://www.youtube.com/watch?v=EEflh5_IWwo&list=PL1EP7zsxqFT_gb2QbK-fxf4tjRumHLqlX&index=2>
